### ISSUE TITLE
A11y_Visual Studio Code Client_Search_ExtensionScoll to Top_Focus Visible: Keyboard focus is not visible on "Scroll to top" button when we are navigating using the tab key

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#203411]
[([Check out Accessibility Insights!](https://accessibilityinsights.io/))]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #Keyboard; #WCAG2.4.7; #AIlimited; #FocusVisible; 

### Environment and OS details:
Visual Studio Code insider version 1.86.0
Microsoft Windows 11 Enterprise 22H2 Build 22621.3007

### Repro Steps:

1. Open Visual Studio Code Insiders editor
2. Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3. Using Tab key navigate to the "More" option and select different options( "Show installed Extension, Show outdated extension, Show popular Extension,Etc")
4. Click on any installed extension
5. Tab till scroll to top button.
6. Verify whether the Keyboard focus is not visible on "Scroll to top" button or not.

### Actual Result:
Keyboard focus is not visible on "Scroll to top" button when we are navigating using the tab key.

### Expected Result:
Keyboard focus should be visible on "Scroll to top" button when we are navigating using the tab key.

### User Impact:
Keyboard users will face difficulties is the focus is not visible on scroll to top button.

### Attachment:

![Bug_203411_Focus Visible](https://github.com/microsoft/vscode/assets/124244478/fe6d5b9a-e4df-4697-a272-e257cc187bc1)

**Comment by meganrogge** [comment]
that button does not receive tab focus.

**Comment by meganrogge** [comment]
one can scroll using arrow keys, so closing this as it is keyboard navigable that way.
